qualang_tools/plot/plot.py

```diff
@@ -97,6 +98,7 @@ def plot_demodulated_data_1d(
     title: str = None,
     amp_and_phase: bool = True,
     fig=None,
+    fit=None,

```

Comment:
Adding a new variable in the middle is a breaking change; it should at least be noted in the changelog.

Reply:
I put it as the last input so that it doesn't break, thanks!